openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -7384,7 +7395,7 @@ def representative_dataset_gen():
     if output_full_integer_quant_tflite:
         try:
             print(f'{Color.REVERCE}Full Integer Quantization started{Color.RESET}', '=' * 51)
-            converter = tf.lite.TFLiteConverter.from_saved_model(model_output_path)
+            converter = get_TFLiteConverter()
             converter.experimental_new_quantizer = use_experimental_new_quantizer
             converter._experimental_disable_per_channel = not use_per_channel
```

Comment:
There are some models that cannot be converted by `tf.lite.TFLiteConverter.from_keras_model(model)`. Therefore, do not modify here or call `tf.lite.TFLiteConverter.from_saved_model(model_output_path)` if an exception occurs during conversion. The `output_saved_model` flag must then be `True`.